include/zephyr/dt-bindings/pinctrl/rpi-pico-rp2040-pinctrl.h

```diff
@@ -25,6 +25,11 @@
 #define RP2_PIN_NUM_POS 4
 #define RP2_PIN_NUM_MASK 0x1f
 
+#define RP2_GPIO_OVERRIDE_NORMAL 0
```

Comment:
These defines don't appear to be used.

Reply:
Normal, Invert, High, Low are the 4 possible values (they are named this way in the rp2 SDK). The intent is that users setting this field would use the options instead of the numeric constants. I could document this in the yaml file instead of using the numbers.